tests/integration-test/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "voxel51-platform-sdk-integration-test",
+  "version": "1.0.0",
+  "description": "Server and integration test suite to permit analytic developers local testing of images prior to platform upload.",
+  "main": "./server/index.js",
+  "repository": "https://github.com/voxel51/platform-sdk",
+  "author": "Voxel51",
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
+    "busboy": "^0.3.1",
+    "joi": "^14.3.1",
+    "ramda": "^0.26.1",
+    "uuid": "^3.3.2"
+  },
+  "scripts": {
+    "start": "node server/index.js",
+    "clean": "rm ./server/storage/*",
```

Comment:
This needs to be refactored into a script because you need to read the new `config.js` file to determine where the storage directory is